src/main/webapp/sass/_core.scss

```diff
@@ -52,7 +52,12 @@ $synapse-tertiary-color-palette: (
   900: #2f2814,
 );
 
-$full-root-panel-height: calc(100vh - 93px - 130px);
+$page-header-min-height: 93px;
+$footer-height: 160px;
```

Comment:
This is not used anywhere (the footer height is dynamic). It's changed since it's last been set, so while this could change again, it at least looks better now.